src/FE/src/app/pages/users-page/users-page.component.html

```diff
@@ -0,0 +1,104 @@
+              {{ 'USERNAME' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let element">
+            <span>
+              {{ element['username'] }}
+            </span>
+            </td>
+          </ng-container>
+
+          <!-- Change password button column -->
+          <ng-container matColumnDef="changePassword">
+            <th mat-header-cell *matHeaderCellDef>
+              {{ 'CHANGE_PASSWORD' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let row">
+              <button mat-icon-button (click)="showChangePassword(row)">
+                <mat-icon>lock_reset</mat-icon>
+              </button>
+            </td>
+          </ng-container>
+
+          <!-- Delete button column -->
+          <ng-container matColumnDef="delete">
+            <th mat-header-cell *matHeaderCellDef>
+              {{ 'DELETE' | translate }}
+            </th>
+            <td mat-cell *matCellDef="let row">
+              <button mat-icon-button (click)="onDelete(row)">
+                <mat-icon>delete</mat-icon>
+              </button>
+            </td>
+          </ng-container>
+
+          <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+          <tr mat-row *matRowDef="let row; columns: displayedColumns" (click)="selectRow(row)"></tr>
+        </table>
+      </div>
+    </div>
+
+    <div *ngIf="userForm" class="newUserContainer" [formGroup]="createUserForm">
+      <p>{{ 'NEW_USERS' | translate }}</p>
+      <mat-form-field>
+        <mat-label>{{ 'USERNAME' | translate }}</mat-label>
+        <input matInput formControlName="username">
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>{{ 'PASSWORD' | translate }}</mat-label>
+        <input matInput formControlName="password" type="password">
+      </mat-form-field>
+
+      <mat-form-field>
+        <mat-label>{{ 'REPEAT_PASSWORD' | translate }}</mat-label>
+        <input matInput formControlName="repeatPassword" type="password">
+      </mat-form-field>
+
+      <button mat-raised-button (click)="onSave()">
+        <mat-icon>
+          add
+        </mat-icon>
+        {{ 'ADD_USER' | translate }}
+      </button>
+
+      <div *ngIf="showResetPasswordFields" class="resetClass">
+        <form [formGroup]="createUserForm">
+          <p>{{ 'CHANGE_PASSWORD' | translate }}</p>
+          <div class="newRepeatClass">
+            <mat-form-field>
+              <mat-label>{{ 'NEW_PASSWORD' | translate }}</mat-label>
+              <input matInput formControlName="newPassword" type="password" class="passwordFields">
+            </mat-form-field>
+            <mat-form-field>
+              <mat-label>{{ 'REPEAT_PASSWORD' | translate }}</mat-label>
+              <input matInput formControlName="repeatPassword" type="password" class="passwordFields">
+            </mat-form-field>
+          </div>
+            <button mat-raised-button (click)="onChangePassword()">
```

Comment:
tilføj `[disable]="<FORMGROUP>.invalid"`. på den måde skal felterne være udfyldt korrekt for at kunne trykke reset.

ps. erstat <FORMGROUP> med navnet på den formgroup, der bruges her